### PR TITLE
update ramp_down_info FAQ EN

### PR DESCRIPTION
**Pull request by MikeMcC399**
Simplification of English text for ENF deactivation.

This follows on from the suggestion in https://github.com/corona-warn-app/cwa-website/pull/3457#discussion_r1162449312.

The English text is better aligned to the German text and thereby also simplified.

https://www.coronawarn.app/en/faq/results/#ramp_down_info reads:

"After April 30, 2023, the Corona-Warn-App will deactivate the ENF (Exposure Notification Framework)."

### CODE REVIEW

**Comment by MikeMcC399** [comment]
@larswmh 

Is this PR being considered?

**Comment by larswmh** [comment]
@MikeMcC399 

> Is this PR being considered?

Yes, we have forwarded this and are waiting for feedback.

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
One suggestion, else this looks good from my side.

File: src/data/faq.json
Comment (MikeMcC399) [reply src/data/faq.json:52]: @Ein-Tim 

I don't want to expand the request for change because I expect that would delay the approval again.

**Comment by MikeMcC399** [comment]
@larswmh 

I noticed when I compared the text of 

- https://www.coronawarn.app/de/faq/results/#ramp_down_info (DE) with
- https://www.coronawarn.app/en/faq/results/#ramp_down_info (EN)

that the English text is not a translation of the German text. The English text is shorter and it is missing parts.

Perhaps there should be an attempt to make sure that the whole of the English text is a correct translation of the German text?

**Comment by larswmh** [comment]
@MikeMcC399 

> I noticed when I compared the text of
> 
> * https://www.coronawarn.app/de/faq/results/#ramp_down_info (DE) with
> * https://www.coronawarn.app/en/faq/results/#ramp_down_info (EN)
> 
> that the English text is not a translation of the German text. The English text is shorter and it is missing parts.
> 
> Perhaps there should be an attempt to make sure that the whole of the English text is a correct translation of the German text?

We will raise this internally, as well as the suggestion by @Ein-Tim

File: src/data/faq.json
Comment (Ein-Tim) [reply src/data/faq.json:52]: @MikeMcC399 

I'm sorry should my comment cause further delay! I should have raised this sooner, but this PR slipped through, sorry.

Comment (MikeMcC399) [reply src/data/faq.json:52]: @Ein-Tim 

No problem. Because of your comment I reviewed the whole of the article and realised that there is a bigger issue. So we can leave it to @larswmh to check and decide with whomever makes decisions on text content about what to change, if anything.

**Review by larswmh — APPROVED** [review]
Thanks for the PR @MikeMcC399 and for your contribution @Ein-Tim! I will merge this once the necessary check is done

**Pull request merged**
